src/components/CommentAdder.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from "react";
+import { postNewComment } from "../utils/utils";
+
+function CommentAdder({ articleId, setComments }) {
+    const [newComment, setNewComment] = useState({
+        username: "grumpy19",
+        body: "",
+    });
+
+    const handleFormSubmit = (event) => {
+        event.preventDefault();
+        postNewComment(articleId, newComment)
+            .then((postedCommentFromDB) => {
+                setComments((currentComments) => {
+                    return [postedCommentFromDB, ...currentComments];
+                });
+            })
+            .catch((err) => {
+                console.log(err);
```

Comment:
so this console log doesn't really serve the user - maybe perhaps you could set a boolean state to indicate if there is an error, and then conditionally render a message to the user when there is an error?